tests/lib/state_test.py

```diff
@@ -240,6 +240,39 @@ def test_absent_cannot_remove_absent(self):
         obj2 = state.RouteEntry(routes[0])
         assert not obj1.is_match(obj2)
 
+    def test_state_iface_routes(self):
```

Comment:
- The test is checking the Status class, I would suggest not grouping it with the tests of `RouteEntity`.
- Naming: Try to express what it actually tests in the name, so we know what is covered. I think that in this case, the scenario is something like: `test_routes_indexed_on_two_distinct_ifaces`